linux-tools/readline/Makefile

```diff
@@ -24,8 +24,8 @@
 ###########################################################################
 
 TARGETS = rltest fileman
-LDFLAGS = ../shlib/libreadline.so.6.2 -lncurses
-CFLAGS = -O2 -I../ -DREADLINE_LIBRARY
+LDFLAGS = /usr/lib*/libreadline.so.6.2 -lncurses
+CFLAGS = -O2 -I /usr/include/readline  -DREADLINE_LIBRARY 
```

Comment:
Here version is hard coded as libreadline.so.6.2 and compilation will fail for different versions. 
libreadline.so can be used as it will be a soft link to respective installed version